packages/deployment-service-ui/src/components/pages/pipelines/show.tsx

```diff
@@ -7,40 +7,39 @@ import { Button, Label, Loader, StatusIndicator, Table, Intent } from '@reapit/e
 import React, { useEffect, useState } from 'react'
 import { Link, useParams } from 'react-router-dom'
 import {
-  DeploymentStatus,
   PipelineModelInterface,
   PipelineRunnerModelInterface,
   TaskModelInterface,
 } from '@reapit/foundations-ts-definitions'
 import { Pagination } from 'nestjs-typeorm-paginate'
 import { PipelineTask } from '@/components/task'
 
-const pipelineStatusToIntent = (status: DeploymentStatus): Intent => {
+const pipelineStatusToIntent = (status: string): Intent => {
```

Comment:
Why no enum now?

Reply:
yeaaaa I don't want to install aws-sdk on the frontend for CodeBuild namespace 🙄